packages/protocol/foundry.toml

```diff
@@ -22,10 +22,11 @@ remappings = [
 
 no_match_test = "skip"
 
-# `BLS12Passthrough.sol` is excluded, because it's tested in the celo-blockain repo as 
-# described here: https://github.com/celo-org/celo-monorepo/pull/10240
-# `Random.sol` is excluded, but I'm not sure why. It was already excluded so I'm leaving it here.
-no_match_path = "*test/{BLS12Passthrough.sol,RandomTest.sol}"
+# 1. `BLS12Passthrough.sol` is excluded, because it's tested in the celo-blockain repo as described here: https://github.com/celo-org/celo-monorepo/pull/10240
+# 2. `Random.sol` is excluded, but I'm not sure why. It was already excluded so I'm leaving it here.
```

Comment:
I think `RandomTest` is excluded because they are a mock and not really a test. But not sure why it should be excluded as it's not a `.t.sol` file. Have you tried what happened if it's removed from this list?

If no truffle tests use it, it can be moved to the `test-sol` folder for good. It's a pitty as it'll get deprecated anyway.